dashboard/consts.py

```diff
@@ -52,3 +52,10 @@
 # ray.dashboard.modules.snapshot.snapshot_head.RayActivityResponse.
 # Example: "your.module.ray_cluster_activity_hook".
 RAY_CLUSTER_ACTIVITY_HOOK = "RAY_CLUSTER_ACTIVITY_HOOK"
+
+# Whether head node deploy raylet process or not.
+RAY_RAYLETLESS_HEAD_NODE = env_bool("RAY_RAYLETLESS_HEAD_NODE", False)
```

Comment:
Should this also be a flag? I think `ray start --head --no-raylet` should be good.